MapboxNavigation/DayStyle.swift

```diff
@@ -121,6 +121,8 @@ open class DayStyle: Style {
         NavigationMapView.appearance().trafficModerateColor     = .trafficModerate
         NavigationMapView.appearance().trafficHeavyColor        = .trafficHeavy
         NavigationMapView.appearance().trafficSevereColor       = .trafficSevere
+        
+        UIApplication.shared.statusBarStyle = statusBarStyle ?? .default
```

Comment:
What about this property https://github.com/mapbox/mapbox-navigation-ios/blob/51470ad955bd4b41cfe9adfd6451911576a0b5f5/MapboxNavigation/DayStyle.swift#L48 ?

Reply:
Is there anything missing here? We must store the property but wait with applying because of the day/night switch.